### DISCUSSION TITLE
Tips on how to model the production of windows.

### DISCUSSION THREAD

**Discussion by nico-or** [frePPLe/frepple#639] [Q&A]
Hi, I'm new to Frepple and were trying to figure out how to model a glass-cutting process.

A little about out process:
- We manufacture insulated glass units on a make-to-order operation.
- Most process capacities are modeled as linear-meters (cutting, sanding and sealing edges, etc).
- Raw material is modeled either as glass sheets units (they vary in sizes) or as square-meters (total sheets x sheet area).

The planning workflow is basically:
- Individual purchase orders come on a regular basis.
- Individual orders are grouped in production batches (about 4 to 5 per day)
- Production batches are build using another software that solves the 2D packing problem.
- A production batch a can be summarized with 2 metrics, total linear-meters and total sheets / square meters of raw material.

My initial idea was to input the final production batches as Frepple's purchase orders. Each an order for 2 products, total linear-meters, and total square-meters.

Some concerns:
- The individual orders (thus, clients, order priorities and due dates) are lost inside the production batch.
- Each process consumes linear-meters but then it has to... pass them to the next process? (cut, then sand, then seal the same perimeters).
- The initial process should consume the totality of the glass surface to reflect the inventory movements.
- The duality of raw material and process capacity units (square meters vs linear meters).

--- 

I would like to hear if there is a better way to model such a process or if another kind of MRP software/strategies are better suited for these kind requirements.

Best regards.

> Nicolás

**Comment by jdetaeye** [comment]
My 2 cents:
- Frepple can plan for the production of the individual orders.  You'll need a separate operation for each sales order.
- The individual, sales-order-specific operations could consume from 2 resources: one that has a "linear" capacity limit and one that has a "surface" capacity limit.   This capacity limit is used for rough-cut capacity planning.   (Variation:  to simplify things, you can take one of these as the leading capacity constraint and use only that one for rough-cut planning)
- Your specialized "packing" / "nesting" program will organize some the individual sales-order-specific manufacturing orders and group them.    You can choose whether or not you feed the results back into the frepple plan (as confirmed manufacturing orders). The feedback can be used to plan the subsequent steps.    If the rough-cut planning is sufficient to manage your follow-up steps, you could skip this feedback.
